.github/workflows/ci.yml

```diff
@@ -5,13 +5,11 @@ name: CI
 on:
   # This will run when any branch or tag is pushed
   push:
-    # standalone is an old branch containing a fully functional pypiserver
-    # executable, from back in the day before docker & a better pip.
-    branches-ignore:
-      - standalone
+    branches:
+      - "master"
```

Comment:
@mplanchard just double checking for sanity, this should still respect the ignore that was set on `standalone`, correct? 🙃